### ISSUE TITLE
Sync to FTP

### ISSUE THREAD

**Issue by chr0max** [MacPass/MacPass#793]
One thing I really miss from the windows keepass is the open and save of a file directly from/to a ftp server. So you can simply use a single file on multiple devices with your own ftp space.

**Comment by chr0max** [comment]
Yes I also thought that, but on Keepass it works flawlessly. On Mac I have Mountain Duck and it works, but not without problems. Sometimes for example Macpass says "the file is changed overwrite?". But there is nothing changed, maybe Mountain Duck changed his cached version or so. (By the way, mountain duck connects on login too fast, before the internet connection is active and throws an error etc...). 

Is's not perfect and I presume that nearly every user today has multiple devices and is interested in an easy way to have access to his passwords without any bigger cloud solution with privacy issues. 

Maybe some sort of plugin system would be helpful and every user who need this functionality, could install it. After some time there could be a very interesting set of plugins. I know that this is a huge bunch of work, but MacPass is the best macclient I found and would deserve it. :-)

**Comment by chr0max** [comment]
by the way, on iOS I have the same problem. There is no keepass client with ftp connection (at least I found no one) and I have to manually sync the files with a ftp client on the device. It's far from perfect. So Ifffff there would be a golden-5-star-ultra-awesome-multi-device-solution, it would include a iOS client in the big strategy password ecosystem.

**Comment by gagarine** [comment]
FTP is dead (at least for me ;) )
What can be done is allow to execute a file/command on save. Then a scp command can be used to copy the file on a server trough SSH. Using ssh key so you don't have to write your password on the command. Easy and secure.

**Comment by gagarine** [comment]
@mstarke yes, certainly.
